ui/components/component-library/text/text.js

```diff
@@ -35,7 +35,7 @@ export const ValidTags = [
 export const Text = React.forwardRef(
   (
     {
-      variant = TEXT.BODY_MD,
+      variant = TEXT_VARIANTS.BODY_MD,
```

Comment:
As per our convention. Use general design system constants unless for storybook controls or propTypes

```suggestion
      variant = TEXT.BODY_MD,
```